app/views/collections_mailer/manage_access_granted_email.html.erb

```diff
@@ -0,0 +1,14 @@
+<%= render Emails::SalutationComponent.new(user: @user) %>
+
+<p>You have been invited to be a Manager of the <%= @collection.title %> collection in the
+   Stanford Digital Repository. Managers have the ability to manage global settings for the collection,
+   manage Depositors and other participants, and edit any deposits in the collection.</p>
+
+<p>To visit the collection where deposits are made, and for more information about SDR repository services,
+   please visit http://<%= Settings.host %>.</p>
```

Comment:
Is there a way to avoid putting http:// in the URL and use https:// instead? I assume this is because it needs to work with localhost?

Reply:
No, that is an oversight. Fixed.